Source/WebDriver/SessionHost.h

```diff
@@ -145,6 +145,10 @@ class SessionHost final
 #elif USE(INSPECTOR_SOCKET_SERVER)
     Function<void(bool, std::optional<String>)> m_startSessionCompletionHandler;
     std::optional<Inspector::ConnectionID> m_clientID;
+    bool m_isRemoteBrowser { true };
```

Comment:
I think we can share above m_isRemoteBrowser with USE(GLIB).

Reply:
Currently `USE(INSPECTOR_SOCKET_SERVER)` is enabled in win and playstation ports.
Either is fine for win, but I think true is better for playstation.

Reply:
I don't understand. Why problem happens for PlayStation port if the initial value is `true`. I think `SessionHost::connectToBrowser` set the variable `false`.

Reply:
> Why problem happens for PlayStation port if the initial value is true.

No problems have occurred if the initial value is true. I said "I think true is better for playstation."
This is because PlayStation's WebDriver only supports remote automation.

Therefore, I think that the initial value as socket port should be `true`, and only `false` if the browser is launched in Windows.
What do you think?

Reply:
^ Sorry. Please ignore the above comment.
The `SessionHost::isRemoteBrowser()` commonality will fix this part too.